deeplabcut/pose_estimation_tensorflow/predict_supermodel.py

```diff
@@ -3,344 +3,17 @@
-                            )
-
-                        PredicteData["frame" + str(ind).zfill(strwidth)].append(pred)
-
-                batch_ind = 0
-                inds.clear()
-            else:
-                batch_ind += 1
-        elif counter >= nframes:
-            # in case we reach the end of the video
-            if batch_ind > 0:
-                preds = []
-                for scale_id, batched_frames in enumerate(multi_scale_batched_frames):
-                    D = predict.predict_batched_peaks_and_costs(
-                        test_cfg,
-                        batched_frames,
-                        sess,
-                        inputs,
-                        outputs,
-                    )
-
-                    preds.append(D)
-
-                ind_start = inds[0]
-                for i in range(batchsize):
-                    ind = ind_start + i
-                    if ind >= nframes:
-                        break
-                    PredicteData["frame" + str(ind).zfill(strwidth)] = []
-                    for scale_id in range(len(scale_list)):
-                        if i >= len(preds[scale_id]):
-                            pred = []
-                        else:
-                            pred = preds[scale_id][i]
-                        if pred != []:
-                            pred = _project_pred_to_original_size(
-                                pred, old_shape, frame_shapes[scale_id]
-                            )
-                        PredicteData["frame" + str(ind).zfill(strwidth)].append(pred)
-
-            break
-
-        counter += 1
-        pbar.update(1)
-
-    cap.close()
-    pbar.close()
-
-    for k, v in PredicteData.items():
-        if v != []:
-            PredicteData[k] = _average_multiple_scale_preds(v, scale_list, num_kpts)
-
-    PredicteData["metadata"] = {
-        "nms radius": test_cfg.get("nmsradius", None),
-        "minimal confidence": test_cfg.get("minconfidence", None),
-        "sigma": test_cfg.get("sigma", 1),
-        "PAFgraph": test_cfg.get("partaffinityfield_graph", None),
-        "PAFinds": test_cfg.get(
-            "paf_best", np.arange(len(test_cfg["partaffinityfield_graph"]))
-        ),
-        "all_joints": [[i] for i in range(len(test_cfg["all_joints"]))],
-        "all_joints_names": [
-            test_cfg["all_joints_names"][i] for i in range(len(test_cfg["all_joints"]))
-        ],
-        "nframes": nframes,
-    }
-
-    return PredicteData, nframes
-
+import deeplabcut
+from deeplabcut.modelzoo.api import SpatiotemporalAdaptation
 
 def video_inference_superanimal(
     videos,
     superanimal_name,
-    scale_list=[],
-    invert_color=False,
-    videotype="avi",
-    destfolder=None,
-    batchsize=1,
-    robust_nframes=False,
-    allow_growth=False,
-    init_weights = "",
-    customized_test_config="",
+    scale_list = [],
+    videotype = '.mp4',
+    video_adapt = False,
+    pcutoff = 0.1
```

Comment:
let's run black afterwards on these files; there should be no spacing on the sides of the = sign.